parser/src/parser/ParserOutput.java

```diff
@@ -0,0 +1,33 @@
+package parser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+public class ParserOutput {
+
+  List<Stack<String>> inputStackStates;
```

Comment:
would recommend to make these fields private and generate getters/settres for them or use lombok for cleaner code